src/main/kotlin/com/sourcegraph/cody/listeners/CodySelectionInlayManager.kt

```diff
@@ -41,17 +41,22 @@ class CodySelectionInlayManager(val project: Project) {
     val startOffset = event.newRange.startOffset
     val endOffset = event.newRange.endOffset
     if (startOffset == endOffset) {
-      // Don't show if there's no selection.
-      return
+      return // Don't show if there's no selection.
```

Comment:
@steveyegge Shouldn't `isCodyUIHintsEnabled` be also checked somewhere in this code block?
I fail to see any place where it is actually used, but perhaps I miss something obvious

Reply:
Damn, good catch. I had it at one point, and a merge killed it. Fixed.